src/components/dev-dot-content/mdx-components/mdx-hcp-callout/index.tsx

```diff
@@ -0,0 +1,101 @@
+import Image from 'next/image'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import Text from 'components/text'
+import StandaloneLink from 'components/standalone-link'
+import { HcpProductSlug } from 'types/products'
+import { productSlugsToNames } from 'lib/products'
+import { MdxHcpCalloutProps, SolutionOption } from './types'
+import patternApplications from './img/applications.png'
+import patternInfrastructure from './img/infrastructure.png'
+import patternNetworking from './img/networking.png'
+import patternSecurity from './img/security.png'
+import s from './mdx-callout.module.css'
+
+const SOLUTION_PRODUCTS_MAP: Record<SolutionOption, HcpProductSlug[]> = {
+	applications: ['waypoint'],
+	infrastructure: ['packer'],
+	networking: ['consul'],
+	security: ['boundary', 'vault'],
+}
```

Comment:
(no action needed) future 💭 I wonder if we want to put the product vertical / solutions area in the product data. I could see it being useful elsewhere at some point.

Reply:
💯 I agree I think this data can be useful in a more global/shared spot